packages/terra-worklist-data-grid/src/WorklistDataGrid.module.scss

```diff
@@ -35,4 +38,49 @@
   .worklist-data-grid-row:nth-child(odd) {
     background-color: var(--terra-worklist-data-grid-row-striped-background-color, #f0f0f0);
   }
+
+  .worklist-data-grid-cell-selected {
+    background-color: var(--terra-worklist-data-grid-cell-selected-background-color, #cbe7fa);
+    box-shadow: var(--terra-worklist-data-grid-cell-selected-box-shadow, inset 0 0 0 1px #0092e0);
+  }
+
+  .worklist-data-grid-row-selected {
+    background-color: var(--terra-clinical-data-grid-row-selected-background-color, #cbe7fa);
+    box-shadow: var(--terra-clinical-data-grid-row-selected-box-shadow, inset 0 1px 0 0 #0092e0, inset 0 -1px 0 0 #0092e0);
+  }
+
+  .icon-container {
+    align-items: center;
+    display: flex;
+    height: 100%;
+    justify-content: center;
+  }
+
+  .icon {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    display: inline-block;
+    height: var(--terra-clinical-data-grid-row-selection-icon-height, 1.143rem);
+    width: var(--terra-clinical-data-grid-row-selection-icon-width, 1.143rem);
+  }
+
+  .row-selectable .icon {
+    background: var(--terra-clinical-data-grid-row-selection-icon-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="rgba(28, 31, 33, 0.04)" d="M42,4H6C4.9,4,4,4.9,4,6v36c0,1.1,0.9,2,2,2h36c1.1,0,2-0.9,2-2V6C44,4.9,43.1,4,42,4z"/><path fill="rgba(28, 31, 33, 0.40)" d="M42,4c1.1,0,2,0.9,2,2v36c0,1.1-0.9,2-2,2H6c-1.1,0-2-0.9-2-2V6c0-1.1,0.9-2,2-2H42 M42,0H6C2.7,0,0,2.7,0,6v36 c0,3.3,2.7,6,6,6h36c3.3,0,6-2.7,6-6V6C48,2.7,45.3,0,42,0L42,0z"/></svg>'));
+  }
+
+  .row-selected .icon {
```

Comment:
Are we using these SVG images or the input field?  I cannot find where this "row-selected" class is used in the code.